### DISCUSSION TITLE
A simple editor

### DISCUSSION THREAD

**Discussion by JoshPaulie** [community/community#8267] [Mobile] [answered]
The app has such a robust issue manager, but nothing for actual code.

The ability to edit readmes is a good first start. Editing could be as simple as editing an issue comment. My reliance on 3rd party apps for these features stinks.

**Comment by eliperkins** [comment]
Thanks for the feedback @JoshPaulie ! Editing code is definitely on our radar, and we want to make sure that we do it right! Starting with editing READMEs is a great idea.

I've added your notes to our internal tracking issue about editing code within the app. I'll talk to the team about this some more!

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @JoshPaulie! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code
